conf/up4.bess

```diff
@@ -192,10 +193,11 @@ linkMerge::Merge() \
                                         {'attr_name':'qer_id', 'num_bytes':4}, \
                                         {'attr_name':'far_id', 'num_bytes':4}],\
                                 entries=parser.table_size_pdr_lookup):noGTPUDecap \
-    -> preQoSCounter::Counter(name_id='ctr_id', check_exist=True, total=parser.max_sessions)
+    -> preQoSCounter::Counter(name_id='ctr_id', check_exist=True, total=parser.max_sessions) \
+    -> preQosMeasure::QosMeasure(flag_attr_name="buffer_flag", entries=parser.table_size_qos_measure)
```

Comment:
Ideally do not insert this module by default. Make it opt-in. If we are keeping it, then turn off the histogram feature, unless asked. We definitely see huge impact on fastpath performance. Confirm if you observe the same.